src/servant/tt_001/F84.c

```diff
@@ -403,7 +411,7 @@ void func_us_80171864(Entity* self) {
         temp_v0 = self->ext.ghost.unk8A;
         self->ext.ghost.unk8A = -temp_v0;
```

Comment:
`self->ext.ghost.unk8A *= -1;` matches PSP better I think

Reply:
Oh, man, that's fantastic.  I was trying to figure that out last night for a while, but I couldn't get it to stop extending the sign flag.  That's the winner for that problem.